DEVELOPMENT.md

```diff
@@ -14,34 +14,17 @@ For developers who want to contribute or modify PECOS:
    ```sh
    make venv
    ```
-   <details>
-   <summary>or manually set up a Python virtual environment for develop of this project's code.</summary>
 
-   On Linux/Mac:
+   or manually set up a Python [virtual environment using uv](https://docs.astral.sh/uv/getting-started/installation/) for develop of this project's code.
 
-   ```sh
-   python -m venv .venv
-   source .venv/bin/activate
-   pip install -U pip setuptools
-   pip install -r python/quantum-pecos/requirements.txt
-   make metadeps
-   pre-commit install
-   ```
-
-   On Windows:
+   To do so, navigate to the root of this project and run:
 
    ```sh
-   python -m venv .venv
-   .\venv\Scripts\activate
-   pip install -U pip setuptools
-   pip install -r python/quantum-pecos/requirements.txt
-   make metadeps
-   pre-commit install
+   python -m pip install --upgrade uv
```

Comment:
`uv` is best installed systemwide, I'd point people to https://docs.astral.sh/uv/getting-started/installation/ instead of suggesting `pip install` here.